doc/governance.rst

```diff
@@ -20,95 +20,107 @@ the project community.
 Roles And Responsibilities
 ==========================
 
+We distinguish between contributors, core contributors, and the technical
+committee. A key distinction between them is their voting rights: contributors
+have no voting rights, whereas the other two groups all have voting rights,
+as well as permissions to the tools relevant to their roles.
```

Comment:
Must we mention the Communication Teams, the Contributor Experience Teams and Maintainers as well?

Reply:
maintainers, comms and contributor experience team are all under "core contributors" here. The "two" here are the core contributors and the TC.